### ISSUE TITLE
Majuscules accentuées et ligature dans le thésaurus Prisme

### ISSUE THREAD

**Issue by daniel-menard** [daniel-menard/prisme#295]
(suite de #187, déjà évoqué par téléphone)

Maintenant qu'on a le thésaurus en version riche, certains termes peuvent avoir des majuscules accentuées (par exemple `Éducation` ou `Chef d'État`) ou des ligatures (par exemple : `Œdipe`, `Sœur`, etc.)

Bien que ce soit l'orthographe correcte des mots, Olivier a fait remarquer que ces caractères pouvaient poser problème car ils sont difficiles à saisir sur un clavier standard. 

Plus exactement, ils sont difficiles à saisir sur un *clavier azerty en français sous Windows* : sur mac ou linux ou bien avec d'autres disposition de claviers - azerty belge, qwerty québécois, [dvorak](https://fr.wikipedia.org/wiki/Disposition_Dvorak) ou [bépo](http://bepo.fr/wiki/Accueil) par exemple - cela ne pose pas de soucis...

Par ailleurs, cela peut également poser des problèmes de mots-clés en doublons avec des logiciels qui ne gèrent pas correctement les équivalences de caractères et considèrent que `e`, `é`, `E` ou `É` sont des caractères différents.

Lors du travail effectué sur la [version 2.2 du thésaurus Prisme](#187), j'avais donc commencé à neutraliser ces caractères en les remplaçant par le caractère de base équivalent ([translittération](https://fr.wikipedia.org/wiki/Transcription_et_translitt%C3%A9ration)) mais finalement, je suis revenu sur cette décision.

En effet, ces caractères ne sont un soucis *que pour les "vieux" logiciels documentaires*, ils ne posent aucun problème dans docalist : que ce soit en saisie (avec les lookups) ou en recherche, on peut taper indifféremment `education`, `éducation`, `Education` ou `Éducation`, on obtiendra toujours le bon mot-clé (en saisie) et les bonnes réponses (en recherche).

Un thésaurus étant un outil transversal, indépendant des logiciels utilisés, je pense qu'il n'y a pas lieu de "niveler par le bas" à cause des limites actuelles des logiciels utilisés actuellement.

Le fait d'avoir un thésaurus en forme riche, avec les termes correctement orthographiés est un investissement pour l'avenir : pour le référencement, pour le web sémantique, etc. Et dernier point, cela permet d'appliquer un correcteur orthographique sans avoir tous les "E sans accent" qui sortent en erreur !

Pour régler le problème des majuscules accentuées lors de l'export vers les logiciels du réseau Prisme, je propose d'avoir une seconde table (appelons-la `thesaurus-prisme-2.2-pauvre`) construite automatiquement (via un script) à partir de la table "officielle" `thesaurus-prisme-2.2`.

Dans docalist, c'est la table riche qui serait utilisée mais lors de l'export, on utiliserait cette nouvelle table afin de générer des mots-clés en minuscules mais sans majuscules accentuées.

Qu'en pensez-vous ?

**Comment by ocormont** [comment]
ok pour l'export, la solution me convient parfaitement.

Un doute par rapport à l'import de notice : Si les notices importées dans Docalist sont sans majuscule accentué ou ligature, sont elles transformées ? 
Si non, à l'affichage, cela risque de faire des saisies différentes, même si cela est le même terme pour Docalist

**Comment by Nicolas-L** [comment]
Je suis d'accord... mais je ne suis pas sûr à 100% que ce soit nécessaire de faire une V.2.2 "pauvre".

Je peux me tromper mais : 

- Pour les problèmes liés au clavier, les producteurs ne sont pas concernés, la com langage pourra (et devra) faire l'effort d'accentuer sur les nouveaux termes quand elle en créera.

- La question se pose pour les logiciels qui ne gèrent pas les équivalences mais, s'il en existe encore dans le réseau, ils ont depuis toujours importé en tout majuscule non accentué. S'ils se mettent aujourd'hui à importer en minuscules cela créera de toute façon des doublons dans leur thésaurus. Cela peut évoluer certes, mans les faits, je pense que le format riche ne servira que très peu pour les imports de notices (dans les bdd locales). Il nous servira avant tout pour l'affichage sur le site (et pour l'AFERTES :-).

Je pencherais donc plus pour que nous nous contentions de proposer une et une seule forme riche complète dans les labels (accents, ligature, maj accentuées...) qui cherche à donner le maximum d'informations sur les termes.

Mais bien sûr s'il s'agit d'un script ou d'une manip très simple... je ne suis pas contre la proposition

**Comment by ocormont** [comment]
Je pensais que cela avait été réglé quand j'ai intégré les deux dernières semaines de mise à jour.
Je n'ai pas eu de remonté par le réseau, mais cela ne devrait pas tarder...

**Comment by Nicolas-L** [comment]
Petite remarque et petit effet de bord sur les majuscules accentuées : la liste qui se déploie dans l'aide à la saisie pour l'indexation (ex : Domaines ou Thésaurus) présente les termes par ordres alphabétiques. Cependant lorsqu'une majuscule est accentuée (ex : Âge ou Éducation), le terme en question passe en dernier de sa lettre (si rien n'a été saisi dans le champ).

Dans Domaine Prisme : "Éducation" se présente après "Vieillissement"
Dans Thésaurus Prisme : "Échec" se présente après "EHPAD"
